### PR TITLE
TakeArguments java file is committed

### PR DESCRIPTION
**Pull request by ArathySuresh29**

### CODE REVIEW

File: TakeArguments.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class TakeArguments{
+public static void main(String[] args){
+if (args.length==1){
```
Comment (raaaimund) [inline TakeArguments.java:3]: If there is a single argument, then everything works as expected (args.length is 1) but what happens if there are multiple arguments (args.length = 3)? Will the output be the argument, or "No Arguments are given"?

File: TakeArguments.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class TakeArguments{
```
Comment (raaaimund) [inline TakeArguments.java:1]: Please format your code! Which editor do you use? IntelliJ or Notepad or something different?

File: TakeArguments.java (hunk 1 above)
Comment (ArathySuresh29) [reply TakeArguments.java:3]: if args.length=3 then the output will be "No Arguments given!" .

File: TakeArguments.java (hunk 2 above)
Comment (ArathySuresh29) [reply TakeArguments.java:1]: Notepad is used as the editor.

Comment (raaaimund) [reply TakeArguments.java:1]: I think IntelliJ would be a much better option from now on. Please use this editor. If you need any help setting it up or if you have any other questions, please do not hesitate to ask.

**Comment by raaaimund** [comment]
Please fix the condition that also the first argument gets printed if you pass multiple arguments and after that, you can merge the pull request.

File: TakeArguments.java (hunk 2 above)
Comment (ArathySuresh29) [reply TakeArguments.java:1]: Hi Raimund, Me and Vishnu are working in a team. He modified some files so please review and merge the pull request.

**Comment by raaaimund** [comment]
Very good @ArathySuresh29 and @vishnusaimanchineella. An awesome decision to work as a team, and that will save you a lot of time for your final year project if you split the work from now on. You can merge the pull request and start the next assignment.

**Review by raaaimund — APPROVED** [review]
Very good @ArathySuresh29 and @vishnusaimanchineella. An awesome decision to work as a team and that will save you a lot of time for your final year project if you split the work from now on. You can merge the pull request and start the next assignment.

**Pull request merged**